envoy/api/v2/route/route.proto

```diff
@@ -197,6 +212,13 @@ message WeightedCluster {
     // :ref:`cluster_metadata <envoy_api_field_route.RouteAction.cluster_metadata>` from the
     // enclosing :ref:`envoy_api_msg_route.RouteAction`.
     core.Metadata cluster_metadata = 7;
+
```

Comment:
Are we going to rollback cluster_metadata?

Reply:
Yes I think @yuval-k is going to do in a follow up. Right @rodaine ?